src/lib.rs

```diff
@@ -28,8 +28,6 @@ pub const TLS_SERVER_ROOTS: &[TrustAnchor] = &[
    * Subject: CN=DigiCert TLS ECC P384 Root G5 O=DigiCert, Inc.
    * Label: "DigiCert TLS ECC P384 Root G5"
    * Serial: 13129116028163249804115411775095713523
-   * MD5 Fingerprint: d3:71:04:6a:43:1c:db:a6:59:e1:a8:a3:aa:c5:71:ed
```

Comment:
Having these changes in a separate commit means that the tests don't pass in the previous commit, right?

I'd probably make a prefix commit that removes the MD5 and SHA1 fingerprints (which is most of the changes here, so it makes it easier to review the remaining changes), and then keep the remaining changes together with the changes to the codegen test.

Reply:
> I'd probably make a prefix commit that removes the MD5 and SHA1 fingerprints (which is most of the changes here, so it makes it easier to review the remaining changes), and then keep the remaining changes together with the changes to the codegen test.

Sounds good. I will rework accordingly :+1:

Reply:
I landed a prefix commit to remove the MD5/SHA1 fingerprints and included the regeneration in the same commit as the updated generator code.